lib/src/graphql_schemas/update_user.graphql

```diff
@@ -0,0 +1,15 @@
+mutation UpdateTransaction(
```

Comment:
I just noticed this, isn't this supposed to be `UpdateUser` mutation?

Reply:
This file should have been deleted. It was moved to bin/src/graphql/update_user.graphql

I will remove in the next PR, thanks.